pkgs/development/python-modules/sphinx-autobuild/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, livereload
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autobuild";
+  version = "2020.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vbaf4vrxahylyp8zrlw5dx1d2faajp926c3pl5i1wlkp1yll62b";
+  };
+
+  # No tests included.
+  doCheck = false;
+
+  propagatedBuildInputs = [ sphinx  livereload ];
+
+  meta = with lib; {
+    description = "Rebuild Sphinx documentation on changes, with live-reload in the browser.";
```

```suggestion
    description = "Rebuild Sphinx documentation on changes, with live-reload in the browser";
```

See https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes